code/DDSCodeTester.cpp

```diff
@@ -3830,4 +3830,32 @@ void dds_usecase_examples()
         //!--
     }
 
+    {
+        //CONF-MEMORY-QOS-PUBSUB
+
+        ResourceLimitsQosPolicy resource_limits;
+        //The ResourceLimitsQosPolicy is default constructed with max_samples = 5000
```

Comment:
In general
```suggestion
        // The ResourceLimitsQosPolicy is default constructed with max_samples = 5000
```

Reply:
Add and space between `//` and the first letter of the comment. When it's said in general, it means that the comment applies to the entire PR. Please check that this format applies to all comments.